docker/README.md

```diff
@@ -0,0 +1,129 @@
+# SETTING UP THE DOCKER DEV ENVIRONMENT
+
+`make` and `docker compose` work together to simplify the process of setting up a dev environment and running `fpan`, specifically in development. Docker is currently not being used in production.
+
+Core Arches (and thus Django) runs in the `arches` container defined in [docker-compose.yml](./docker-compose.yml). Django/Arches and `fpan` commands run inside this container. Postgres, Elastic Search, and RabbitMQ run in their own containers.
+
+`legiongis/arches#dev/6.2.x-hms-cli`, `legiongis/arches-extensions`, and `fpan` repos are mounted as volumes to the `arches` docker service so they can all be edited. Open these directories on your host machine to make code changes. You should see your changes to `fpan` in the browser after reloading the page. Editing arches and arches-extensions has not been tested.
```

Comment:
Quick edit in `arches` was reflected immediately in the devserver, so I'll update this comment.